src/hooks/useFocusWithin/useFocusWithin.ts

```diff
@@ -186,15 +190,15 @@ function useFocusEvents({
         (event: React.FocusEvent) => {
             if (
                 document.activeElement !== event.target &&
-                (event.relatedTarget === null ||
+                ((!mobile && event.relatedTarget === null) ||
```

Comment:
It might be better to disable `useFocusWithin` in `Select` on mobile devices.

Reply:
@ValeraS, when I disable the hook, the tests for closing the select fail because onClose is not being called. It looks like the hook is needed on mobile

<img width="638" alt="Screenshot 2025-02-05 at 16 03 32" src="https://github.com/user-attachments/assets/cd57042d-ddb7-419a-ad11-81b17a1c4ff3" />

Reply:
We can replace click on body with `await userEvent.click(screen.getByTestId(SelectQa.SHEET));` in tests

Reply:
Yes, but userEvent.click(...) doesn't trigger [onTransitionEnd](https://github.com/gravity-ui/uikit/blob/feat/select-close-on-mobile/src/components/Sheet/SheetContent.tsx#L148) we need to close Sheet. The test worked because (with useFocusWithin) click triggered blur. I'll research this moment

Reply:
why are these changes still here?

Reply:
Sorry, missed it after rebase